.env.example

```
@@ -2,6 +2,10 @@ NEXT_PUBLIC_SUPABASE_URL=
 NEXT_PUBLIC_SUPABASE_ANON_KEY=
 SUPABASE_SERVICE_ROLE_KEY=
 RESONANCE_PROJECT_ID=
+# auto | optional | required — production must be required
+RESONANCE_AUTH_MODE=required
+# development | production — set production on the live host
+RESONANCE_DEPLOY_STAGE=production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RESONANCE_DEPLOY_STAGE key should go before the RESONANCE_PROJECT_ID key

(UnorderedKey)